spark/dl/src/main/scala/com/intel/analytics/bigdl/optim/DistriOptimizer.scala

```diff
@@ -522,6 +566,7 @@ object DistriOptimizer {
         cached.map(_._4), // criterions
         cached.map(_._5), // states
         cached.head._2.clone(), // a tensor buffer
+        cached.head._2.clone(), // a tensor buffer for elastic difference
```

Comment:
Looks like elastic difference in a node is the same, I think we can just keep one copy in node to save memory just as what we did for weight.